### DISCUSSION TITLE
Am I stupid?

### DISCUSSION THREAD

**Discussion by goldendog64** [QuestCraftPlusPlus/QuestCraft#549] [Q&A]
In the description it syas that skyblock/oneblock works best, however I dont see an options to import maps and I cant find a single tutorial on how to do it in the files.
Am I stupid or is it just not added yet because the "open worlds folder" is not working either.

**Comment by KendrickLamar2003** [comment]
There is a "saves" folder in the root of .minecraft. Drop the world folder there.
